brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-927 Fix: Gradient - Type and Angle field misalign issue in multi-button

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-927

### Screenshots

- Fix - https://d.pr/i/uRSP5b

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop button block.
- Go to Style > Background > Gradient and see is it misaligns or not.
- See screenshot.

### Checklist:
